mask/eolearn/mask/cloud_mask.py

```diff
@@ -329,32 +312,40 @@ def _dilate_all(self, data):
 
         return data
 
-    def _ssim_stats(self, bands, is_data, win_avg, var, nt_rel, sigma):
+    def _ssim_stats(
+        self,
+        bands: np.ndarray,
+        is_data: np.ndarray,
+        local_avg: np.ndarray,
+        local_var: np.ndarray,
```

Comment:
would sure be nice to know what these two actually area. Perhaps not the time (since we're still moving stuff around), but at some point we'll need to add docstrings